lib/github-glue.ts

```diff
@@ -11,6 +11,7 @@ export interface IPullRequestInfo {
     baseCommit: string;
     baseOwner: string;
     baseRepo: string;
+    commits?: number;
```

Comment:
Oh crap, I forgot about that in #181... I called the same field `patchCount`... Do you have a strong opinion on the name (I guess the GitHub response data calls it `commits`...)? I will be happy to change my PR.

Reply:
On second thought, I don't think I can rely on this field to be set, and it is also more accurate to call it "patch count" ("commits" would also include merge commits, after all), so I'll just keep my `patchCount`...

Reply:
Yes, I followed the GitHub api field name.  The field is not set for `GitHubGlue::getOpenPRs()` responses but it is for `getPRInfo()`.  Yes it does include merges but those will be rejected at the front end soon.